resources/dicts/thoughts/alive/general.json

```
@@ -503,6 +1948,24 @@
             "starclan"
         ]
     },
+    {
+        "id": "gen_to_dead_parent",
+        "thoughts":[
+            "Heard r_c humming a tune in {PRONOUN/m_c/poss} dream {PRONOUN/m_c/subject} remembered {PRONOUN/r_c/object} humming to {PRONOUN/m_c/object} in the nursery",
```

```suggestion
            "Heard r_c humming a tune in {PRONOUN/m_c/poss} dream that {PRONOUN/m_c/subject} remembered r_c humming in the nursery",
```
Just cutting down on some of the pronoun usage, flip flopping between two pronoun sets without interjecting some names makes it confusing, esp if both cats use the same pronouns